app/lib/constants/site_config.rb

```diff
@@ -10,7 +10,8 @@ module SiteConfig
         placeholder: ""
       },
       require_captcha_for_email_password_registration: {
-        description: "People will be required to fill out a captcha when they're creating a new account in your community",
+        description: "People will be required to fill out a captcha when
+          they're creating a new account in your community",
```

Comment:
Good fix! This was on my to-fix list-- running `--no-verify` is too many keystrokes. 😎